components/HistoryTable.qml

```diff
@@ -52,6 +54,22 @@ ListView {
             + translationManager.emptyString;
     }
 
+    function lookupPaymentID(paymentId) {
```

Comment:
Looking at the AddressBook code, it's indexed by... index, but there's a getRowID. What is that ?